Platform/ApollolakeBoardPkg/CfgData/CfgData_PcieRp.dsc

```diff
@@ -27,11 +27,11 @@
   # !BSF SUBT:{PCIERP_TMPL:5 : 0x8E}
 
   #                                      Power       : Reset
-  # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:0 :  0x0021318A  : 0x0021618E}
-  # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:1 :  0x0021118A  : 0x0020D18A}
-  # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:2 :  0x0000018B  : 0x0020F18A}
+  # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:0 :  0x6021318A  : 0x0021618E}
+  # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:1 :  0x8021118A  : 0x0020D18A}
+  # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:2 :  0x4000018B  : 0x0020F18A}
   # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:3 :  0x0000018B  : 0x0022518A}
-  # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:4 :  0x0000018B  : 0x0000018B}
+  # !BSF SUBT:{PCIERP_CTRL_PIN_TMPL:4 :  0x2000018B  : 0x0000018B}
```

Comment:
I have double-checked these against the Leaf Hill CRB schematic and all looks valid and correct to me.

Reply:
Thanks for confirmation.